install_hooks.py

```diff
@@ -16,25 +59,60 @@ def get_init_scripts(config):
     elif 'bsd' in sys.platform or 'dragonfly' in sys.platform:
         data_files = data_files + "\nusr/local/etc/rc.d = bin/rc.d/*"
     else:
-        raise "Unsupported platform, sorry"
-        data_files = []
+        raise Exception("Unsupported platform, sorry")
+
     config['files']['data_files'] = data_files
 
+    for line in config['files']['data_files'].split('\n'):
+        line = line.strip().split('=')
+        print("Installable directories/files: %s" % line)
+
+    # Check Alignak recommended user existence
+    if not user_exists('alignak'):
```

Comment:
You are making alignak software depending on a system user. I don't think it is a good idea. devs are forced here to have such a user. A warning is ok, but it may not prevent a user from setuping Alignak